docs/user/discover.asciidoc

```diff
@@ -275,7 +270,24 @@ image:images/discover-maps.png[Map containing documents]
 [[share-your-findings]]
 === Share your findings
 
-To share your findings with a larger audience, click *Share* in the *Discover* toolbar. For detailed information about the sharing options, refer to <<reporting-getting-started,Reporting>>.
+To share your findings with a larger audience, click *Share* in the *Discover* toolbar.
+For detailed information about the sharing options, refer to <<reporting-getting-started,Reporting>>.
+
+[float]
+[[alert-from-Discover]]
+=== Generate an alert
+
+From *Discover*, you can create an alert to periodically
```

Comment:
Terminology wise, you don't create an alert. You create a rule, which does stuff when conditions are met (which are tracked via alerts):
```suggestion
From *Discover*, you can create a rule to periodically
```